temporal/api/cloud/cloudservice/v1/service.proto

```diff
@@ -118,4 +118,78 @@ service CloudService {
             get: "/api/v1/regions/{region}",
         };
     }
+
+    // Gets all known incoming services
+    rpc GetIncomingServices(GetIncomingServicesRequest) returns (GetIncomingServicesResponse) {
+	option (google.api.http) = {
+            get: "/api/v1/nexus/incomingservices",
```

Comment:
Can we add a `/by-id` or `/id` suffix to this URL just in case we want to allow looking up by name eventually?

Reply:
since I am following what we do for users, I've kept the same format (which doesn't have by-id or id either..). @anekkanti , thoughts?

Reply:
yes, am inclined to leave as-is as well. can always add suffix later if needed